packages/babel-preset/non-standard-plugins.js

```diff
@@ -38,6 +42,20 @@ module.exports = function shopifyNonStandardPlugins(options = {}) {
         {loose: true},
       ],
     );
+
+    if (transformRuntime) {
+      // When compiling TypeScript with only babel-loader, extra helpers are
+      // inlined in each source module. This normally adds some bulks to files,
+      // but `transform-runtime` converts these blocks into common helper imports
+      // that webpack can collate into shared assets.
+      plugins.push([
+        '@babel/plugin-transform-runtime',
+        {
+          version: require('@babel/plugin-transform-runtime/package.json')
```

Comment:
Doesn't have to happen in this PR, but I'd be keen to see how options that I missed work out:
* [`useESModules`](https://babeljs.io/docs/en/babel-plugin-transform-runtime#useesmodules)
* [`regenerator`](https://babeljs.io/docs/en/babel-plugin-transform-runtime#regenerator) - I think most Shopify apps have a global regenerator? So maybe this plugin doesn't have to do so much.
* [`corejs`](https://babeljs.io/docs/en/babel-plugin-transform-runtime#corejs) - maybe doesn't make sense until everything's on core-js@3

Reply:
`useESModules` looks really promising